joomla/joomla-cms
# [Hathor] Adding icon to com_associations

### Summary of Changes
Adding specific icons for header and menu for com_associations
Icon based on the side by side symbol
### Testing Instructions
Create a multilingual site.
Display Multilingual associations.

Before patch: no icons
After patch, you should get this:

![screen shot 2017-04-02 at 07 57 02](https://cloud.githubusercontent.com/assets/869724/24584906/b734a9f6-177d-11e7-9ea8-358672997609.png)

No idea if a patch containing binaries will work or not with patchtester.
If not, please pick 
https://github.com/joomla/joomla-cms/compare/staging...infograf768:hathor_assoc_icon?expand=1#diff-6f72cf792495bdb18d4015f139d3ad4c
and place manually in:
` administrator/templates/hathor/images/header/icon-48-assoc.png `

As well, pick https://github.com/joomla/joomla-cms/compare/staging...infograf768:hathor_assoc_icon?expand=1#diff-03d3757f3c0056ac27f76c906c2417d3

and place in
` administrator/templates/hathor/images/menu/icon-16-assoc.png`
